nsds_lab_to_nwb/metadata/metadata_manager.py

```diff
@@ -69,6 +71,14 @@ def common_check(self):
         '''
         if 'subject' not in self.metadata_input:
             self.metadata_input['subject'] = {}
+            self.metadata_input['subject']['subject_id'] = self.animal_name
+
+        # fix subject weight unit - always 'g' in our case
+        subject_metadata = self.metadata_input['subject']
+        if 'weight' in subject_metadata:
+            weight = str(subject_metadata['weight'])
+            if weight[-1] != 'g':
```

Comment:
I'm not clear on how this field is created, but maybe it's better to check `if 'g' not in weight`:? For instance, that would catch if a trailing space was entered after the 'g' or something like that.

Reply:
Good point, pushed a fix commit!